listenbrainz/db/stats.py

```diff
@@ -0,0 +1,55 @@
+"""This module contains functions to insert and retrieve statistics
+   calculated from Google BigQuery into the database.
+"""
+
+import sqlalchemy
+import ujson
+from listenbrainz import db
+
+
+# XXX(param): think about the names of these stats variables
```

Comment:
which variables are you referring to?

Reply:
Ugh. DB table names should always be singular. Can we still change this?

Reply:
Yeah, I did this before I became aware of the singular names thing. Can be easily changed, I'll make an issue for it.  https://tickets.metabrainz.org/browse/LB-201